consensus/replay_test.go

```diff
@@ -238,26 +232,25 @@ func TestWALCrashBeforeWritePropose(t *testing.T) {
 
 func TestWALCrashBeforeWritePrevote(t *testing.T) {
 	for _, thisCase := range testCases {
-		testReplayCrashBeforeWriteVote(t, thisCase, thisCase.prevoteLine, types.EventStringCompleteProposal())
+		testReplayCrashBeforeWriteVote(t, thisCase, thisCase.prevoteLine)
 	}
 }
 
 func TestWALCrashBeforeWritePrecommit(t *testing.T) {
 	for _, thisCase := range testCases {
-		testReplayCrashBeforeWriteVote(t, thisCase, thisCase.precommitLine, types.EventStringPolka())
+		testReplayCrashBeforeWriteVote(t, thisCase, thisCase.precommitLine)
 	}
 }
 
-func testReplayCrashBeforeWriteVote(t *testing.T, thisCase *testCase, lineNum int, eventString string) {
+func testReplayCrashBeforeWriteVote(t *testing.T, thisCase *testCase, lineNum int) {
 	// setup replay test where last message is a vote
 	cs, newBlockCh, voteMsg, walFile := setupReplayTest(t, thisCase, lineNum, false)
-	types.AddListenerForEvent(cs.evsw, "tester", eventString, func(data types.TMEventData) {
```

Comment:
uh  I don't think so. These tests are overdue for some refactoring but the point was to make sure we're at a particular point in the consensus state machine before